sdks/python/apache_beam/utils/annotations.py

```diff
@@ -90,6 +90,8 @@ def inner(*args, **kwargs):
       message = '%s is %s' % (fnc.__name__, label)
       if label == 'deprecated':
         message += ' since %s' % since
+        if extra_message is not None:
+          message += '. ' + extra_message
       message += '. Use %s instead.'% current if current else '.'
```

Comment:
Thanks, I added the missing space.
As for the location of the `extra_message`, I put it as the next to last because I figured that if there is a current version to provide, no `extra_message` (other instructions) will likely be needed. I figured it is in the deprecating process before there is a `current` suggestion that `extra_message` is useful.